Content.Shared/Chemistry/Solution.cs

```diff
@@ -17,6 +17,7 @@ public class Solution : IExposeData, IEnumerable<Solution.ReagentQuantity>
         // Most objects on the station hold only 1 or 2 reagents
         [ViewVariables]
         private List<ReagentQuantity> _contents = new List<ReagentQuantity>(2);
+        public List<ReagentQuantity> Contents => _contents;
```

Comment:
This really needs to be exposed as a read-only interface, like `IReadOnlyList<T>`. You do not want people to try and manually add things to the list, they need to be using the AddReagent function.